web/lib/handlers/order.js

```diff
@@ -0,0 +1,33 @@
+const { asyncify } = require('async');
+const mastermind = require('../mastermind');
+const Order = require('../models/order');
+
+const get = params => // eslint-disable-line no-unused-vars
+  mastermind([
+    // eslint-disable-next-line no-unused-vars
+    asyncify(async (req, reply, cb) => {
+      const { userId } = req.user.id;
```

Comment:
shouldn't it be `const userId = req.user.id` ? To my knowledge, req.user.id is already a string, using destructuration doesn't help